content/concepts/multiplex/yamux.md

```diff
@@ -1,15 +1,34 @@
 ---
-title: "yamux"
+title: "Yamux"
 description: "yamux is a multiplexing protocol designed by Hashicorp."
 weight: 180
 ---
 
-[yamux](https://github.com/hashicorp/yamux) is a multiplexing protocol designed by [Hashicorp](https://www.hashicorp.com/).
-yamux offers more sophisticated flow control than mplex, and can scale to thousands of multiplexed streams over a
-single connection.
+## What is Yamux?
+
+[Yamux](https://github.com/hashicorp/yamux) is a multiplexing protocol designed
+by [Hashicorp](https://www.hashicorp.com/) for Go, built over TCP.
+Yamux provides a high-level interface for working with SCTP streams. It is inspired
+by [SPDY](https://en.wikipedia.org/wiki/SPDY), but it is not interoperable with it.
+
+Yamux offers more sophisticated flow control than [mplex](mplex) while also supporting
+backpressure, which is a mechanism that helps to prevent data from being sent faster
+then it can be processed.
+
+Backpressure allows the receiver of a data stream to specify an offset
+to which the sender can send data. This offset increases as the receiver process the data,
```

Comment:
This paragraph should be rephrased. Backpressure in general is just a mechanism to prevent one peer to overwhelm a peer that's slow consuming the data.
Sending the offset is _one_ concrete option to implement backpressure.